assets/src/edit-story/components/canvas/canvasProvider.js

```diff
@@ -68,11 +69,21 @@ function CanvasProvider({ children }) {
         clearEditing();
       }
 
-      if (evt.metaKey) {
+      // Skip the focus that immediately follows mouse event.
+      // Use the reference to the latest element because the events come in the
+      // sequence in the same event loop.
+      if (lastSelectedElementId.current === elId && evt.type === 'focus') {
```

Comment:
Might want to check `lastSelectedElementId.current` is set here.

Reply:
`===` gets me that.